packages/ckeditor5-engine/src/view/view.ts

```diff
@@ -75,7 +75,6 @@ export default class View extends Observable {
 	public readonly domRoots: Map<string, HTMLElement>;
 
 	declare public isFocused: boolean;
-	declare public isSelecting: boolean;
```

Comment:
Breaking change?

Reply:
Nope, it's just a `declare` that wasn't used anywhere.